### PR TITLE
Fix dead link after rearrange english documentation.

### PR DESCRIPTION
**Pull request by reyoung**

### CODE REVIEW

File: doc/howto/cmd_parameter/index_en.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+```eval_rst
+..  _cmd_line_index:
```
Comment (luotao1) [inline doc/howto/cmd_parameter/index_en.md]: 名字后面加en来区分中英文

File: doc/tutorials/rec/ml_dataset_en.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+```eval_rst
+..  _demo_ml_dataset:
```
Comment (luotao1) [inline doc/tutorials/rec/ml_dataset_en.md]: 名字后面加en来区分中英文

Comment (reyoung) [inline doc/tutorials/rec/ml_dataset_en.md]: Done.

File: doc/howto/cmd_parameter/index_en.md (hunk 1 above)
Comment (reyoung) [inline doc/howto/cmd_parameter/index_en.md]: Done.

**Pull request merged**